build/config_dev.go

```diff
@@ -0,0 +1,158 @@
+//go:build dev
+// +build dev
+
+package build
+
+import (
+	"fmt"
+	"strings"
+
+	btclogv1 "github.com/btcsuite/btclog"
+	"github.com/btcsuite/btclog/v2"
+)
+
+const (
+	resetSeq = "0"
+	boldSeq  = "1"
+	faintSeq = "2"
+	esc      = '\x1b'
+	csi      = string(esc) + "["
+
+	callSiteOff   = "off"
+	callSiteShort = "short"
+	callSiteLong  = "long"
+)
+
+// LogConfig holds logging configuration options.
+//
+//nolint:lll
+type LogConfig struct {
+	Console *consoleLoggerCfg `group:"console" namespace:"console" description:"The logger writing to stdout and stderr."`
+	File    *LoggerConfig     `group:"file" namespace:"file" description:"The logger writing to LND's standard log file."`
+}
+
+// DefaultLogConfig returns the default logging config options.
+func DefaultLogConfig() *LogConfig {
+	return &LogConfig{
+		Console: &consoleLoggerCfg{
+			LoggerConfig: LoggerConfig{
+				CallSite: callSiteOff,
```

Comment:
can we set the default to `callSiteShort` for `dev` cfg? think this will make debugging itest much easier!

Reply:
yeah def! good idea

Reply:
done :)